ui/app/components/secret-engine/configure-aws.ts

```diff
@@ -15,22 +16,22 @@ import errorMessage from 'vault/utils/error-message';
 import type LeaseConfigModel from 'vault/models/aws/lease-config';
 import type RootConfigModel from 'vault/models/aws/root-config';
 import type Router from '@ember/routing/router';
-import type Store from '@ember-data/store';
+import type StoreService from 'vault/services/store';
 import type VersionService from 'vault/services/version';
 import type FlashMessageService from 'vault/services/flash-messages';
 
 /**
  * @module ConfigureAwsComponent is used to configure the AWS secret engine
  * A user can configure the endpoint root/config and/or lease/config.
- * For enterprise users, they will see an additional option to config WIF attributes in place of IAM attributes. 
```

Comment:
interesting my linter doesn't pick up these extra spaces. Did you notice them or were they caught by a linter?

Reply:
This was prettier removing them on save!